msp/mspimplsetup_test.go

```diff
@@ -121,6 +147,20 @@ func TestCAValidation(t *testing.T) {
 		gt.Expect(err).NotTo(gomega.HaveOccurred())
 	})
 
+	t.Run("ExpiredCert", func(t *testing.T) {
```

Comment:
Why do we need this test? It's irrelevant to the code change.

Reply:
Ok, I will delete it, it was my way of checking that there were no issues with the CA